mksh/PKGBUILD

```diff
@@ -1,10 +1,10 @@
 # Maintainer: Masanori Shirayama <mashir43@gmail.com>
 
 pkgname=mksh
-_ver=59
+_ver=59c
```

Comment:
needs "59.c" since "59" > "59c"

Reply:
In next line, `_ver` is converted to correct `pkgver`.

Reply:
oops